api/ssh.py

```diff
@@ -0,0 +1,52 @@
+import json
+from log import MeticulousLogger
+from .base_handler import BaseHandler
+from .api import API, APIVersion
+from config import MeticulousConfig, CONFIG_SYSTEM, SSH_ENABLED
+from ssh_manager import SSHManager
+
+logger = MeticulousLogger.getLogger(__name__)
+
+
+class SSHHandler(BaseHandler):
```

Comment:
We have the config endpoint for everything inside the user config. There is no need to create a new endpoint for every change :)